tests/test_release_packaging.py

```
@@ -0,0 +1,74 @@
+                "class Alcove < Formula",
+                '  homepage "https://example.invalid/internal/alcove"',
+                '  url "https://files.pythonhosted.org/packages/source/a/alcove-search/alcove_search-0.1.0.tar.gz"',
+                '  sha256 "REPLACE_WITH_RELEASE_SHA256"',
+                '  license "MIT"',
+                "end",
+            ]
+        ),
+        encoding="utf-8",
+    )
+
+    with pytest.raises(AssertionError, match="Formula/alcove.rb is not release-safe"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)